enrolments/tests/test_serializers.py

```diff
@@ -235,32 +218,110 @@ def test_enrolment_serializer(self):
-        data["session"] = self.other_session.id
-        serializer = EnrolmentSerializer(self.enrolment, data=data)
+class EnrolmentCreateSerializerTestCase(TestCase):
+    def setUp(self):
+        self.parent_field = Field.objects.create(
+            role=Field.PARENT,
+            name="Internet Access",
+            question="Do you have access to internet",
+            question_type=Field.TEXT,
+            is_default=True,
+            order=1,
+        )
+        self.child_field = Field.objects.create(
+            role=Field.CHILD,
+            name="Gender",
+            question="What is their gender?",
+            question_type=Field.MULTIPLE_CHOICE,
+            is_default=True,
+            order=1,
+        )
+        self.guest_field = Field.objects.create(
+            role=Field.GUEST,
+            name="Relationship",
+            question="What's their relationship to your family?",
+            question_type=Field.MULTIPLE_CHOICE,
+            is_default=True,
+            order=1,
+        )
+        self.session = Session.objects.create(
+            season="Spring",
+            year=2021,
+            start_date=date(2021, 5, 15),
+        )
+        self.class_in_session = Class.objects.create(
+            name="Test Class",
+            session_id=self.session.id,
+            facilitator_id=None,
+            attendance=[],
+        )
+        self.family_data = {
+            "email": "weasleys@theorder.com",
+            "cell_number": "123456789",
+            "address": "12 Grimmauld Place",
+            "preferred_comms": "Owl Post",
+        }
+        self.parent_data = {
+            "first_name": "Molly",
+            "last_name": "Weasley",
+            "information": {f"{self.parent_field.id}": "yes"},
+        }
+        self.children_data = [
+            {
+                "first_name": "Ron",
+                "last_name": "Weasley",
+                "information": {f"{self.child_field.id}": "male"},
+            },
+            {
+                "first_name": "Ginny",
+                "last_name": "Weasley",
+                "information": {f"{self.child_field.id}": "female"},
+            },
+        ]
+        self.guests_data = [
+            {
+                "first_name": "Harry",
+                "last_name": "Potter",
+                "information": {f"{self.guest_field.id}": "friend"},
+            }
+        ]
+
+    def test_family_detail_serializer_create(self):
+        family = dict(self.family_data)
+        family["parent"] = self.parent_data
+        family["children"] = self.children_data
+        family["guests"] = self.guests_data
+
+        data = {
+            "family": family,
+            "session": self.session.id,
+            "preferred_class": self.class_in_session.id,
+            "status": Enrolment.REGISTERED,
+        }
+
+        serializer = EnrolmentCreateSerializer(data=data)
         self.assertTrue(serializer.is_valid())
```

Comment:
since we overrode `validate` as well, it'd be good to throw in a separate test for it to ensure we call the right validators! similar idea to [these tests](https://github.com/uwblueprint/project-read-backend/blob/c7fb3e4c3d3bd25c3aa4c2437f0592f4b5e321a2/registration/tests/serializers/test_family_detail_create.py#L124-L142)